src/github/push.ts

```diff
@@ -37,5 +37,5 @@ export const pushWebhookHandler = async (context: WebhookContext, jiraClient, _u
 	}
 
 	context.log.info("Enqueueing push event");
-	await enqueuePush(payload, jiraClient.baseURL);
+	await enqueuePush(payload, jiraClient.baseURL, context.gitHubAppConfig);
```

Comment:
Does **gitHubAppConfig** need to be added to the **WebhookContext** type?

Reply:
...but you're already using it in the context here.  He's just saying to add it to the type

Reply:
OH, SORRY, haha, thought something else. That disturbed thing really got me. 
I did added the `gitHubAppConfig` to the `WebhookContext` types in file `src/routes/github/webhook/webhook-context.ts`. Not sure I get it right?

Reply:
Closing this threads, since the type is already in the `WebhookContext` type.